types/testingutils/beacon_node_test.go

```diff
@@ -1,12 +1,14 @@
 package testingutils
 
 import (
-	"github.com/stretchr/testify/require"
 	"testing"
+
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/stretchr/testify/require"
 )
 
 func TestBeaconBlockRoot(t *testing.T) {
-	r1, _ := TestingBeaconBlock.HashTreeRoot()
-	r2, _ := TestingBlindedBeaconBlock.HashTreeRoot()
+	r1, _ := TestingBeaconBlockV(spec.DataVersionBellatrix).Root()
```

Comment:
the different naming in the interface is a bit confusing, I assume `HashTreeRoot` and `Root` are the same functionality.